packages/fondue/src/components/Tree/types.ts

```diff
@@ -139,6 +139,7 @@ export type TreeProps = {
     selectedIds?: string[];
     expandedIds?: string[];
     dragHandlerPosition?: DragHandlerPosition;
+    /** @deprecated unused, property was used to at 150 ms delay when dragHandlerPosition is 'none'. but could stop registering trags. */
     enableDragDelay?: boolean;
```

Comment:
Tbh it is way more confusing to keep it but unused.
Since it might be used externally, keep the behavior as it was. But you can change the default behavior when we use the Tree component and it is unset (`enableDragDelay === undefined`).